
int8 axis
float64 angle
float64 actualRate